### PR TITLE
DM-1820 SmartREST static template 143.

### PR DESCRIPTION
**Pull request by MaciekSoftwareAG**

### CODE REVIEW

File: content/reference/smartrest-two-bundle/mqtt-static-templates.md [hunk 1]
```diff
@@ -543,6 +544,21 @@ Removes advanced software items from the list that exists for the device.
 142,docker,3.2.1,nginx,1.6
 ```
 
+<a name="143"></a>
+##### Set supported software types (143)
+
+Set supported software types of the device. Ignores empty elements. Empty list removes c8y_SupportedSoftwareTypes fragment entirely.
```
Comment (rssag) [inline content/reference/smartrest-two-bundle/mqtt-static-templates.md:550]:
```suggestion
Set the supported software types of the device. Ignores empty elements. An empty list removes the `c8y_SupportedSoftwareTypes` fragment entirely.
```

**Review by rssag — CHANGES_REQUESTED** [review]

File: content/reference/smartrest-two-bundle/mqtt-static-templates.md [hunk 2]
```diff
@@ -49,6 +49,7 @@ If a parameter is in square brackets, it is optional.
 + <a href="#140">140,setAdvancedSWName1,AdvancedSWVersion1,AdvancedSWType1,AdvancedSWurl1,sw2,ver2,type2,url2,...</a>
 + <a href="#141">141,appendAdvancedSWName1,AdvancedSWVersion1,AdvancedSWType1,AdvancedSWurl1,sw2,ver2,type2,url2,...</a>
 + <a href="#142">142,deleteAdvancedSWname1,AdvancedSWVersion1,sw2,ver2,...</a>
++ <a href="#143">143,supportedSoftwareType1,supportedSoftwareType2,…</a>
```
Comment (rssag) [inline content/reference/smartrest-two-bundle/mqtt-static-templates.md:52]:
```suggestion
+ <a href="#143">143,supportedSoftwareType1,supportedSoftwareType2,...</a>
```

**Review by rssag — APPROVED** [review]

**Review by l2p- — APPROVED** [review]

**Comment by BeateRixen** [comment]
@MaciekSoftwareAG Can this PR be merged? And to which releases does it apply?

**Comment by MaciekSoftwareAG** [comment]
> @MaciekSoftwareAG Can this PR be merged? And to which releases does it apply?

Yes, I just waiting for the actual new code to be merged first.

**Comment by rssag** [comment]
what are the target versions though?

**Pull request merged**